contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.3.0--3.4.0.sql

```diff
@@ -0,0 +1,41 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
```

Comment:
Can you kindly make sure whether we need one upgrade script for babelfishpg_common extension as well or not?

Reply:
We can bump it up later when we need any SQL changes in the common extension